docs/tls.md

```diff
@@ -1,22 +1,22 @@
-# TLS Support
+# TLS support
```

Comment:
I'm delighted you're not a fan of Title Case

Reply:
:D We aim to Please

Reply:
In other matters -- still need to push edits to proxy-proto.md. Doesn't tls.md provide the same content?

Reply:
yes, there is probably overlap between tls.md and proxy-proto.md. My vote would be to push the complexity into the latter -- not everyone uses aws.